source/tools/testConsistency/ClientApp/app.js

```diff
@@ -0,0 +1,196 @@
+const express = require("express");
+const path = require('path');
+const bodyParser = require('body-parser');
+const http = require('http');
+const restler = require('restler');
+const querystring = require('querystring');
+
+const app = express();
+app.set('view engine', 'ejs');
+app.set('views', path.join(__dirname, 'views'));
+app.use(bodyParser.urlencoded({extended: false}));
+
+app.get('/', function(req, res) {
+  res.render('index.ejs');
+});
+
+app.post('/', function(req, res) {
+  var cardData = req.body.cardData;
+  var cardTemplate = req.body.cardTemplate;
+
+  const formData = querystring.stringify({
+    "data" : cardData,
+    "template" : cardTemplate
+  });
+
+  const optionsForNodeJs = {
```

Comment:
here and below -- we should separate out the per-platform bits of the code from the main parts of the code. where possible, configuration tasks should be separated out as data that is iterated over. this makes debugging, maintaining, and extending functionality easier (e.g. if I wanted to add a new platform, what all would I need to edit to get it working?)